models/DataObject/Traits/ContextPersistenceTrait.php

```diff
@@ -57,8 +57,8 @@ protected function enrichDataRow($object, array $params, ?string &$classId, &$ro
             $context = $object->getContext();
             if (isset($context['containerType']) && ($context['containerType'] === 'fieldcollection' || $context['containerType'] === 'objectbrick')) {
                 $fieldname = $context['fieldname'];
-                $index = $context['index'] ?? null;
-                $row['ownername'] = '/' . $context['containerType'] . '~' . $fieldname . '/' . $index . '/localizedfield~' . $row['ownername'];
+                $row['index'] = $context['index'] ?? 0;
```

Comment:
This is definitely wrong for objectbricks as there we don't have an index.
Previous: "//"
After: "/0/"

Reply:
thx for review, then it's better to keep the index as it was before. if i have more time then i will setup a better docker image for testing.